config/base.json

```diff
@@ -14,8 +14,7 @@
 		"canonical_base_url": "https://developer.hashicorp.com",
 		"max_static_paths": 10,
 		"revalidate": 360,
-		"content_preview_branch": "dev-portal",
-		"beta_product_slugs": ["waypoint", "vault"],
```

Comment:
One idea that we could explore is just enabling waypoint as a beta slug in the 'preview' config. I was going to try this to enable the product for previews in the tutorials repo